packages/manager/apps/container/package.json

```diff
@@ -11,13 +11,13 @@
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
   "scripts": {
-    "build": "vite build",
-    "dev": "vite",
-    "dev:dedicated": "APP=dedicated vite",
-    "dev:hub": "APP=hub vite",
-    "dev:public-cloud": "APP=public-cloud vite",
-    "dev:telecom": "APP=telecom vite",
-    "dev:web": "APP=web vite",
+    "build": "tsc && vite build",
```

Comment:
any reason to run "tsc" before vite ? since vite will transpile typescript to javascript